scheduled-jobs/build/bug-sweeps/4.1/Jenkinsfile

```diff
@@ -0,0 +1,30 @@
+properties(
+    [
+        buildDiscarder(
+            logRotator(
+                artifactDaysToKeepStr: '',
+                artifactNumToKeepStr: '',
+                daysToKeepStr: '',
+                numToKeepStr: '90'
+            )
+        ),
+	disableConcurrentBuilds(),
+    ]
+)
+
+description = ""
+failed = false
+
+b = build job: 'sweep', propagate: false, parameters: [
```

Comment:
I am **not sure** if using the `build job` thing understands to make relative references. This `4.1` job is _supposed to_ call the `sweep` job in this same folder with the `BUILD_VERSION` parameter.

Reply:
It's relative, yeah.

But why not put the parameterized job in the aos-cd-builds folder? I thought we had this dynamic going where the manual jobs went there and the scheduled jobs went here.

Reply:
I have no strong feelings about where it lives.